### PR TITLE
Probability of a Card

### PR DESCRIPTION
**Pull request by willsalz**
New PR against @anthonybishopric's branch.

### CODE REVIEW

File: pandemic/record.go [hunk 1]
```diff
@@ -68,3 +71,28 @@ func (c CityDeck) probabilityOfEpidemic() float64 {
 		return 0
 	}
 }
+
+func (gs GameState) ProbabilityOfCity(cn string) float64 {
```
Comment (anthonybishopric) [inline pandemic/record.go:75]: So we'll probably want to add a `TopOfDeck` section for cards returned from the `Drawn` section after an epidemic. Also there's some conditional probability we'll want to add given the possibility of an outbreak.

File: pandemic/record_test.go [hunk 2]
```diff
@@ -4,6 +4,41 @@ import (
...
+
+	gs := GameState{
+		InfectionDeck: InfectionDeck{
+			Drawn: []InfectionCard{},
+		},
+		InfectionRate: 2,
+	}
+
+	// TODO(wjs) how 2 floating point
+	if prob := gs.ProbabilityOfCity("SomeCity"); prob != 0.04166666666666674 {
```
Comment (anthonybishopric) [inline pandemic/record_test.go:36]: We could test against a rounded value if that makes this more aesthetically pleasing

File: pandemic/record.go (hunk 1 above)
Comment (willsalz) [inline pandemic/record.go:75]: Yep I think I had some notes about that in my commit message…good idea. :)

File: pandemic/record_test.go (hunk 2 above)
Comment (willsalz) [inline pandemic/record_test.go:36]: 👍

**Pull request merged**